SDK/AppCenter/Microsoft.AppCenter.WindowsDesktop/Utils/ApplicationLifecycleHelper.cs

```diff
@@ -89,12 +89,7 @@ static ApplicationLifecycleHelper()
                     .GetRawConstantValue();
             }
 
-
-#pragma warning disable CS0618 // Type or member is obsolete
-            // We need Windows thread ID, not managed
-            var threadId = AppDomain.GetCurrentThreadId();
-#pragma warning restore CS0618 // Type or member is obsolete
-            var hook = SetWinEventHook(EVENT_SYSTEM_MINIMIZESTART, EVENT_SYSTEM_MINIMIZEEND, IntPtr.Zero, hookDelegate, 0, (uint)threadId, WINEVENT_OUTOFCONTEXT);
+            var hook = SetWinEventHook(EVENT_SYSTEM_MINIMIZESTART, EVENT_SYSTEM_MINIMIZEEND, IntPtr.Zero, hookDelegate, 0, 0, WINEVENT_OUTOFCONTEXT);
```

Comment:
I checked we don't get events from other processes, not sure why we were using those integers for before.